doc/sphinx-guides/source/installation/config.rst

```diff
@@ -3860,6 +3860,23 @@ To enable the setting::
   curl -X PUT -d true "http://localhost:8080/api/admin/settings/:DisableSolrFacets"
 
 
+:DisableSolrFacetsForGuestUsers
++++++++++++++++++++++++++++++++
+
+Similar to the above, but will disable the facets for Guest users only. 
+
+:DisableSolrFacetsForAnonymousUsers
```

Comment:
What about `:DisableSolrFacetsForInitialRequests` instead?

Perhaps I'm misunderstanding the paragraph below, but when set to true, people would be affected by this (and not see the facets) the first time they visit a Dataverse installation, right? Unfortunately, these people would get a less rich, facetless experience, if I understand correctly, when this is set to true for the purpose of mostly avoiding loads from bots.

Reply:
Sure, `:DisableSolrFacetsWithoutJsession` (or whatever cApiTaLization) sounds good to me. 😄